.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+jobs:
+  release:
+    if: github.ref == 'refs/heads/master'
```

Comment:
For this repository, we might want to remove this condition because having Semantic Release run on every build in Dependabot or Renovate's automatic PRs is useful to help identify if a patch causes a breaking change. However, I'm not sure if Semantic Release does a dry run on PR branches or just exits out immediately because it's not on master.

@wobrien and @AodhanLP - I'll let you make a call on this 👍

Reply:
Let's try this out by removing this line to see what happens